### ISSUE TITLE
Sensors: Linking sensors to experiments

### ISSUE THREAD

**Issue by aislinnpearson** [Rothamsted-Ecoinformatics/farm_rothamsted#253]
We had a request to link and display sensors associated with specific experiments. I think it is already possible to link a sensor to an experiment by adding two parent assets to the sensor asset:

1. the experiment boundary
2. the plot itself where relevant

But I wasn't sure if that was the best approach given that the experiments are plans. I'm open to ideas

Assuming that this is the best way to link the two assets, it would be nice to have a way of displaying the inked sensor assets either as a sensor tab or as a specific section on the experiment metadata.

**Comment by aislinnpearson** [comment]
As a follow up comment, one of the team at North Wyke asked me if you could load points instead of plots and log records against those in the experiment module. I think this is the wrong approach for sensors associated with experiments as each would assumedly have it's own output data stream and this might or might not be linked to a plot or experiment area, in which case creating sensor assets and linking them to locations is the right way to go about things.

**Comment by aislinnpearson** [comment]
An additional request from Had: once an experiment is complete the sensors must be removed (which in terms of FarmOS would be archived). We also need to be able to associate logs and other observations with the sensors - for example, when they are not working or have been serviced.

**Comment by paul121** [comment]
>  I think it is already possible to link a sensor to an experiment by adding two parent assets to the sensor asset:

An even easier method may be to add the sensor asset to the `plan.asset` field. This field is basically a big bucket that conveys "these assets are associated with this plan." This feels like the better option if not many experiments have sensors.

Another option would be to create a dedicated `plan.sensor` field. This option would make it the easiest to "display" the sensor assets, since there would be a field dedicated to this. I think this option is best if all or many experiments have sensors.

>  it would be nice to have a way of displaying the inked sensor assets either as a sensor tab or as a specific section on the experiment metadata.

With either of these options we can create a dedicated sensor tab or place in the metadata to view these.

One idea that comes to mind is a dedicated "Assets" tab, similar to the plots tab, but that shows all assets in the `plan.asset` field. This could serve other use-cases by having filters to let users see all sensor, land, equipment, etc assets that are associated with the experiment. In general I think it depends how many sensor and other assets you think may be associated with experiment plans - if there are not many, then a dedicate tab may be overkill.

**Comment by paul121** [comment]
@aislinnpearson I'm going to close this, based on our conversation I think we figured it out. Feel free to open if you have any questions!
